docs/config/keys.md

```diff
@@ -112,34 +225,49 @@ prefix.
 Similar in concept to the `phys:` mapping described above, the `raw:` mapping
 is independent of the OS keyboard layout.  Raw codes are hardware and windowing system dependent, so there is no portable way to list which key does what.
 
-To discover these values, you can set [debug_key_events =
-true](lua/config/debug_key_events.md) and press the keys of
-interest.
+To discover these values, you can set [debug_key_events = true](lua/config/debug_key_events.md) in your config, this will log key events in the [debug overlay](lua/keyassignment/ShowDebugOverlay.md) e.g.
+
+<img src="../screenshots/debug-key_event-raw_code.png">
 
 You can specify a raw key value of 123 by using `key="raw:123"` in your config
 rather than one of the other key values.
 
-### Leader Key
+```lua
+local wezterm = require 'wezterm'
+return {
+  keys = {
+    -- These are the same key assignments
```

Comment:
... on a ... keyboard under ...? (above there is `Raw codes are hardware and windowing system dependent,...`)